index.html

```diff
@@ -198,158 +161,238 @@ <h2>Tonic</h2>
-              <input
-                type="text"
-                id="name"
-                name="user_name"
-                maxlength="30"
-                placeholder="your name "
-                required
-                class="form-input"
-              />
+              <a href="#"
+                ><img
+                  class="t-social-media"
+                  src="images/twitter.png"
+                  alt="twitter"
+              /></a>
             </li>
             <li>
-              <label for="mail"></label>
-              <input
-                type="email"
-                id="mail"
-                name="user_mail"
-                placeholder="your email "
-                class="form-input"
-                required
-              />
+              <a href="#"><img src="images/linkedin.png" alt="linkedIn" /></a>
             </li>
             <li>
-              <label for="msg"></label>
-              <textarea
-                id="msg"
-                name="user_message"
-                maxlength="500"
-                placeholder="write something here"
-                class="form-input"
-                rows="5"
-                required
-              ></textarea>
+              <a href="#"><img src="images/medium.png" alt="medium" /></a>
             </li>
             <li>
-              <button class="btn contact-btn" type="submit">Get In Touch</button>
+              <a href="#"><img src="images/github.png" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img class="v-social-media" src="images/vec.png" alt="vector"
+              /></a>
             </li>
           </ul>
-        </form>
+          <button type="button" class="click-btn">Get my resume</button>
+        </section>
+        <section>
+          <ul class="lang">
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Languages</p>
+                <img src="images/arrows-down.png" alt="arrow_bottom" />
+              </div>
+              <ul class="languages">
+                <li>
+                  <img src="images/js.png" alt="JavaScript" />
+                  <a href="#">JavaScript</a>
+                </li>
+                <li>
+                  <img src="images/html..png" alt="CSS" />
+                  <a href="#">HTML</a>
+                </li>
+                <li>
+                  <img src="images/css..png" alt="CSS" />
+                  <a href="#">CSS</a>
+                </li>
+              </ul>
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Frameworks</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Skills</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
```

Comment:
- [ ] As suggested by the previous reviewer, it would be better If you use correct links to display your arrow images.

**Hint: Your arrow icons have as name `Arrow.png` and in your code, you called `arrow`. You can just use replace it with `<img src="images/Arrow.png" alt="arrow_bottom" /> ` to solve the issue**